<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Running Timer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #a7a7a7;
    font-family: 'Segoe UI', sans-serif;
    color: white;
}


QLabel#timer_label {
    font-family: 'Courier New', monospace;
    font-size: 46px;
    font-weight: bold;
    color: #111;
    background-color: #ffffff;
    border-radius: 15px;
    padding: 20px;
    border: 1px solid #999;
    qproperty-alignment: AlignCenter;

   
    background-image: linear-gradient(to bottom,
        #ffffff 0%,
        #ffffff 48%,
        #cccccc 48%,
        #cccccc 52%,
        #ffffff 52%,
        #ffffff 100%);
}


QPushButton {
    background-color: #3c3c3c;
    color: white;
    border: none;
    border-radius: 10px;
    padding: 12px 30px;
    font-size: 14px;
    font-weight: bold;
    letter-spacing: 1px;
}

QPushButton:hover {
    background-color: #505050;
}

QPushButton:pressed {
    background-color: #2e2e2e;
}
</string>
  </property>
  <widget class="QLabel" name="timer_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>491</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="play_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Puase/Resum</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
